db/seeds.rb

```diff
@@ -1,7 +1,128 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
+# This file should contain all the record creation needed to seed the database
+# with its default values.
+# The data can then be loaded with the rails db:seed command (or created
+# alongside the database with db:setup).
 #
 # Examples:
 #
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
+#   movies = Movie.create([{ name: 'Star Wars' },
+#                          { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+bob = User.create(username: 'Bob', email: 'bob@mail.com', password: '123')
+sarah = User.create(username: 'Sarah', email: 'sarah@mail.com', password: '321')
+ivan = User.create(username: 'Ivan', email: 'breku@mail.com', password: 'ur')
+maria = User.create(username: 'Maria', email: 'maria@mail.com', password: 'm!a')
+
+# Resolved bets
+bet1 = Bet.create(bet_description: 'Ninjas vs Pirates',
```

Comment:
If create fails due to validation is fails silently. use `create!` to get an error if creation fails. That was you know if your seed script is failing to do what you think it is doing.